examples/create_udf.ipynb

```diff
@@ -0,0 +1,875 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2227e466-f9e4-4882-9a21-da2b1824b301",
+   "metadata": {},
+   "source": [
+    "# Generate Python UDFs for different cases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86b69eae-351d-45f4-ac16-f3bd8eb2bd42",
+   "metadata": {},
+   "source": [
+    "## Initialization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1f1cc965-ed44-4a7b-9b37-7fd5a611d967",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Suppress some deprecation warnings from langchain one may see on some platfroms\n",
+    "import warnings\n",
```

Comment:
Hmmm, it doesn't seem necessary to have this cell in the example notebook, shall we remove it?

Reply:
I just thought that it would be strange if the first cell would have number 2. Without warnings suppressing there are a lot of deprecation warnings from langchsin. I can delete it.

Reply:
> Without warnings suppressing there are a lot of deprecation warnings from langchsin.

I wonder what's the langchain version on your env. I can try reproducing as well

Reply:
0.0.302
![image](https://github.com/pyspark-ai/pyspark-ai/assets/29755009/519615da-a4de-4ee7-ac41-397cc3385544)